website/pages/docs/secrets/databases/index.mdx

```diff
@@ -152,12 +156,39 @@ for more information.
 
 ## Password Generation
 
-Password generation for both static and dynamic database credentials occurs via
-the `GetPassword()` function. For static credentials, the `SetCredentials()`
-function is also used on the output of `GetPassword().`
-
-Please see the [DB plugin credentials source code](https://github.com/hashicorp/vault/blob/master/sdk/database/dbplugin/database.pb.go)
-for more information.
+Passwords are generated via [Password Policies](/docs/concepts/password-policies).
+Databases can optionally set a password policy for use across all roles for that database.
+In other words, each time you call `vault write database/config/my-database` you can specify
+a password policy for all roles using `my-database`. Each database has a default password
+policy defined as: 20 characters with at least 1 uppercase character, at least 1 lowercase
+character, at least one number, and at least one dash character.
```

Comment:
The first part of the statement uses the number `1`:
```suggestion
character, at least 1 number, and at least 1 dash character.
```

Reply:
:+1: